AutoFlow_OmicsDataHandling/thermo/static/small_ecoli/lexicon.csv

```diff
@@ -0,0 +1,305 @@
+,seed_id
```

Comment:
Do the developers behind the tFBA module provide lexicons for other organisms or is that something that we would have to provide?  Specifically, we would be looking at S. cerevisiae, P. putida, E. coli, and several other gut microbiota strains in the near future.

Reply:
Unfortunately, they only provide lexicons for E. coli, so we would have to work on the other ones ourselves. However, I've been working on a big lexicon right now (because it's also a requirement for BNICE), so this should be easy.

Reply:
That sounds good.  Can you open a new Jira issue for the expanded lexicon?